metrics/network/remote_network/remote-networking-memory-smem.sh

```diff
@@ -135,6 +181,11 @@ function main {
 
 	if [ "$test_rss" == "1" ]; then
 		remote_network_rss_memory
+	elif [ "$test_pss" == "1" ]; then
```

Comment:
rss and pss test are mutually exclusive? if not then you don't need a new option (-t) to run both tests

Reply:
user must be able to run this test with ```-r -p``` options

Reply:
@devimc, that is actually what the script does, the user can run the test with `-r` and `-p`

Reply:
@devimc , I am providing the `-t` option in case that the user wants to run both tests